Typo "a GitHub action"→"a  GitHub Action" (#12432)
MicrosoftDocs/azure-devops-docs: docs/pipelines/ecosystems/github-actions.md

Get started using [GitHub Actions](https://docs.github.com/en/actions) with Azure Pipelines. GitHub Actions help you automate your software development workflows from within GitHub. You can deploy workflows in the same place where you store code and collaborate on pull requests and issues. 

> If you have both Azure Pipelines and GitHub Actions workflows, you might want to trigger a pipeline run from within a GitHub Action. For example, you might have a specific set of pipeline tasks that you want to trigger from your GitHub Actions workflow. You can trigger a pipeline run with the [Azure Pipelines Action](https://github.com/marketplace/actions/azure-pipelines-action).  

Reviewer: ```suggestion
If you have both Azure Pipelines and GitHub Actions workflows, you might want to trigger a pipeline run from within a GitHub action. For example, you might have a specific set of pipeline tasks that you want to trigger from your GitHub Actions workflow. You can trigger a pipeline run with the [Azure Pipelines Action](https://github.com/marketplace/actions/azure-pipelines-action).  
```

## Prerequisites

- A working Azure pipeline. [Create your first pipeline](../create-first-pipeline.md). 
- A GitHub account with a repository. [Join GitHub](https://github.com/join) and [create a repository](https://docs.github.com/en/github/getting-started-with-github/create-a-repo). 

> - An Azure DevOps personal access token (PAT) to use with your GitHub Aactions. [Create a PAT](../../organizations/accounts/use-personal-access-tokens-to-authenticate.md).

Reviewer: ```suggestion
- An Azure DevOps personal access token (PAT) to use with GitHub Actions. [Create a PAT](../../organizations/accounts/use-personal-access-tokens-to-authenticate.md).
```

## Authenticate with Azure Pipelines

> Use a [personal access token](../../organizations/accounts/use-personal-access-tokens-to-authenticate.md) (PAT) to connect your GitHub account to Azure DevOps. You can generate a PAT from within Azure DevOps and then store it as a GitHub secret. Within your GitHub workflow, reference the secret so that your GitHub Actions can authenticate with your Azure DevOps project. 

Reviewer: ```suggestion
Use a [personal access token](../../organizations/accounts/use-personal-access-tokens-to-authenticate.md) (PAT) to connect your GitHub account to Azure DevOps. You can generate a PAT from within Azure DevOps and then store it as a GitHub secret. Within your GitHub workflow, reference the secret so that your GitHub Actions workflow can authenticate with your Azure DevOps project. 
```
